core/src/cluster_info.rs

```diff
@@ -2016,6 +2004,62 @@ impl ClusterInfo {
         Some(packets)
     }
 
+    fn handle_batch_pull_responses(
+        &self,
+        responses: Vec<(Pubkey, Vec<CrdsValue>)>,
+        thread_pool: &ThreadPool,
+        stakes: &HashMap<Pubkey, u64>,
+        epoch_time_ms: u64,
+    ) {
+        type HashMap = std::collections::HashMap<Pubkey, Vec<CrdsValue>>;
```

Comment:
How about something like `ResponseMap`, bit confusing to conflict with the generic `HashMap` name.

Reply:
removed it entirely. it was only a type alias to make things shorter, but i think writing the actual type would be more explicit.